### PR TITLE
#166240980 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by EmyRukundo**
**What does this PR do?**

   Integrate TravisCI with readme badge

 **Description of Task to be completed?**

  this is for running the test and respond 
with the status, if it is passing  or failing  

**What are the relevant pivotal tracker stories?**

#166240980

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.com/andela/tesla-ah.svg?branch=master)](https://travis-ci.com/andela/tesla-ah)
+
+
 Authors Haven - A Social platform for the creative at heart.
```
Comment (deschantkn) [inline README.md]:
```suggestion
Authors Haven - A Social platform for the creative at heart
```

**Review by Diama1 — APPROVED** [review]

**Review by mifeille — APPROVED** [review]

**Review by deschantkn — APPROVED** [review]

**Review by Quantum-35 — CHANGES_REQUESTED** [review]
@EmyRukundo Kindly Squash your commits to one.

**Review by eliemugenzi — APPROVED** [review]

**Comment by rukundoeric** [comment]
Your test script is duplicated and it can't support es6 features

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Your test script is duplicated and it can't support es6 features

**Comment by EmyRukundo** [comment]
> @EmyRukundo Kindly Squash your commits to one.

>@Quantum-35 yes, I did

**Review by rukundoeric — APPROVED** [review]

**Comment by Quantum-35** [comment]
Please fix the merge conflicts

**Comment by EmyRukundo** [comment]
> Please fix the merge conflicts

It's fixed now

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.com/andela/tesla-ah.svg?branch=master)](https://travis-ci.com/andela/tesla-ah)
```
Comment (Quantum-35) [inline README.md]: Change this Link to point to Develop for correct readings.

**Review by Quantum-35 — CHANGES_REQUESTED** [review]

**Comment by Quantum-35** [comment]
Kindly Rebase and delete `package-lock.json` and `node_modules`. Then run yarn install.

**Review by Quantum-35 — APPROVED** [review]

**Pull request merged**